src/SearchByType/SearchByType.test.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import { SearchByType } from './SearchByType';
+import { findByTestAttr, checkProps } from '../../Utils';
+import SearchByTypeReducer from './SearchByTypeReducer';
+import { SET_SEARCH_BY_GENRE } from './SearchByTypeActions';
+
+const setup = (props = {}) => {
+  const component = shallow(<SearchByType {...props} />);
+  return component;
+};
+
+describe('SearchByType component', () => {
+  describe('Check PropTypes', () => {
+    it('should not throw a warning', () => {
+      const expectedprops = {
+        activeSearch: 'title',
+      };
+      const propdError = checkProps(SearchByType, expectedprops);
+      expect(propdError).toBeDefined();
+    });
+  });
+
+  describe('Have Props', () => {
+    let wrapper;
+    beforeEach(() => {
+      const props = {
+        activeSearch: 'title',
+      };
+      wrapper = setup(props);
+    });
+
+    it('should render a activeSort', () => {
+      const activeSort = findByTestAttr(wrapper, 'activeSearch');
+      expect(activeSort.length).toBe(1);
+    });
+  });
+
+  describe('reducer', () => {
```

Comment:
reducer should be tested in separate file, don't mix these types, the same for action, saga function and so on

Reply:
Separated